plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2053,6 +2078,43 @@ static auto maybe_make_debug_time_logger() -> std::optional<decltype(make_debug_
    }
 }
 
+void block_only_sync::schedule() {
+   if (!_pending) {
+      // wait one second to see if we can actually get the block from net plugin before we try to resync from block vault
+      _start_sync_timer.expires_from_now(boost::posix_time::seconds(1));
+      _pending = true;
+      _start_sync_timer.async_wait(
+          app().get_priority_queue().wrap(priority::high, [this](const boost::system::error_code& ec) {
+             if (!ec) {
+                auto id = _impl->chain_plug->chain().last_irreversible_block_id();
+                fc_dlog(_log, "Attempt to resync from block vault");
+                try {
+                  _impl->blockvault->sync(&id, *this);
+                } catch( fc::exception& er ) {
+                   wlog("Attempting to resync from blockvault encountered ${details}; the node must restart to "
```

Comment:
Should be `fc_wlog`.

Reply:
You missed:
line 326 `dlog`.
line 341 `elog`

Not sure if this was intentional.